src/lib/components/workflow/batch-operation-confirmation-modal.svelte

```diff
@@ -17,25 +21,31 @@
     confirm: { reason: string };
   }>();
 
+  $: confirmText = action === 'Cancel' ? 'Confirm' : action;
+  $: placeholder = `${pastTense(action)} from Web UI${
+    $authUser?.email ? ` by ${$authUser.email}` : ''
```

Comment:
What about the case when it's local development and there is no authUser?

Reply:
I think this should be handled with my ternary: `$authUser?.email ? ` by ${$authUser.email}` : ''`, If there is no email, the message will just be `(Terminated|Canceled) from Web UI`. Should we add an IP address or something?

Reply:
durrr my bad